src/components/AddProcessButton/Dialog/Dialog.tsx

```diff
@@ -94,7 +94,7 @@ export function AddProcessButtonDialog({
               definition: values.definitionName,
               _type: 'process',
               item: { _id: itemId, type: itemType },
-              assignedTo: [{ _id: user._id }],
+              assignedTo: [{ _id: user.contacts }],
```

Comment:
this doesn't really make sense to me. Is this what the API requires?

Reply:
yeah, it seems like user._id is something else. the assignedTo field requires a contact id